Have Chromium look in /usr/lib/chromium for Widevine CDM.
Useful for those with chromium-widevine installed from the AUR.

--- a/chrome/common/chrome_paths.cc
+++ b/chrome/common/chrome_paths.cc
@@ -379,8 +379,12 @@ bool PathProvider(int key, base::FilePat
       cur = cur.Append(FILE_PATH_LITERAL("pnacl"));
       break;
 
-#if defined(OS_LINUX) && BUILDFLAG(BUNDLE_WIDEVINE_CDM)
+#if defined(OS_LINUX) && BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
     case chrome::DIR_BUNDLED_WIDEVINE_CDM:
+      cur = base::FilePath(FILE_PATH_LITERAL("/usr/lib/chromium"))
+               .AppendASCII(kWidevineCdmBaseDirectory);
+      if (base::PathExists(cur))
+        break;
       if (!GetComponentDirectory(&cur))
         return false;
 #if !defined(OS_CHROMEOS)
@@ -389,7 +393,7 @@ bool PathProvider(int key, base::FilePat
       cur = cur.AppendASCII(kWidevineCdmBaseDirectory);
 #endif  // !defined(OS_CHROMEOS)
       break;
-#endif  // defined(OS_LINUX) && BUILDFLAG(BUNDLE_WIDEVINE_CDM)
+#endif  // defined(OS_LINUX) && BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
 
 #if defined(OS_LINUX) && !defined(OS_CHROMEOS) && \
     BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
